### ISSUE TITLE
Train-Test Split

### ISSUE THREAD

**Issue by Nishant001-jyo** [Nishant001-jyo/Nishant001-jyo#20]
`X_train, X_test, y_train, y_test = train_test_split(
    X, y, test_size=0.2, shuffle=False
)
`
